### PR TITLE
Migrate from Netlify to Vercel

### PR DESCRIPTION
**Pull request by jordanpowell88**
We want to migrate from deploying via `Netlify` to `Vercel`.

Currently there is a sanity script that is triggered via a cron job that makes a `POST` to Netlify to trigger a build. This PR changes the Netlify Hook url to a new Vercel Hook. 

There is also a bunch of redirect rules in the `netlify.toml` file. This PR migrates those to a `vercel.json` file

I also removed a Netlify plugin that was generating the `sitemap.xml` file and replaced with the nuxtjs module.

It also updates documentation around Netlify and replaces with appropriate Vercel language.

### CODE REVIEW

**Comment by jordanpowell88** [comment]
Here's a Screenshot showing the `POST` to the new Vercel triggers a new build successfully

Screen Shot 2021-12-02 at 2 25 40 PM

**Comment by jordanpowell88** [comment]
This image will be referenced in `CONTRIBUTING.md`
vercel-bot-preview-screenshot

**Comment by cowboy** [comment]
I still see a bunch of (failing) Netlify checks in this PR. Can they be removed?

**Comment by jordanpowell88** [comment]
> I still see a bunch of (failing) Netlify checks in this PR. Can they be removed?

As far as I know, these are configured inside of Netlify and the only way (that I'm aware of) to remove them is to remove them for ALL builds moving forward. So the answer is, yes I can remove them but we should probably not make that switch until everything looks good and we are ready to rip the bandaid off

File: nuxt.config.js [hunk 1]
```diff
@@ -49,7 +49,11 @@ export default {
    ** Plugins to load before mounting the App
    ** https://nuxtjs.org/guide/plugins
    */
-  plugins: ['@/plugins/vue-scrollactive', '@/plugins/sanity-client', {src: '@/plugins/fullstory', mode: 'client'}],
+  plugins: [
+    '@/plugins/vue-scrollactive',
+    '@/plugins/sanity-client',
```
Comment (admah) [inline nuxt.config.js:54]: Are we removing `sanity` from the codebase entirely at this point?

File: nuxt.config.js [hunk 2]
```diff
@@ -72,7 +90,7 @@ export default {
    ** Google Tag Manager
    */
   gtm: {
-    // The env var CONTEXT is set by Netlify and can be 'production', 'deploy-preview', or 'branch-deploy'
+    // The env var CONTEXT is set by Vercel and can be 'production', 'deploy-preview', or 'branch-deploy'
     id:
       (process.env.CONTEXT === 'production' && 'GTM-KNKBWLD') || 'GTM-XXXXXXX',
```
Comment (admah) [inline nuxt.config.js:95]: Has this ENV var been set up in Vercel?

**Review by admah — CHANGES_REQUESTED** [review]
I have a couple of questions before we launch this. Other than those, it looks great.

File: nuxt.config.js (hunk 1 above)
Comment (amirrustam) [reply nuxt.config.js:54]: Due to pricing and functionality, we'll standardize on Contentful for the public website and therefore docs. Though I did prefer many things about Sanity.

Comment (admah) [reply nuxt.config.js:54]: Awesome. I’m great with Contentful! I think just removing the plugin ref should be good then.

**Comment by robertguss** [comment]
@jordanpowell88 this PR looks good to me. Do we have contentful hooked up and ready to go with this PR as well? I have no idea what is needed for that, but just thought I would bring it up before we make the final switch to Vercel.

**Comment by jordanpowell88** [comment]
> @jordanpowell88 this PR looks good to me. Do we have contentful hooked up and ready to go with this PR as well? I have no idea what is needed for that, but just thought I would bring it up before we make the final switch to Vercel.

As far as I know `contentful` is just being used or potentially used for the marketing banner at top. Last time that @amirrustam and I talked he mentioned that we would be just removing the Sanity pieces (since `contentful` would be replacing it) and then we would later add support for `contentful`.

**Review by admah — APPROVED** [review]
LGTM

**Comment by admah** [comment]
@jordanpowell88 this will require a bit of work on the admin side so lets hold on merging until @amirrustam is back and able to help.

**Comment by jordanpowell88** [comment]
Closing this since we no longer plan to make this switch

**Pull request closed without merging**